openedx/core/djangoapps/theming/helpers.py

```diff
@@ -154,6 +155,19 @@ def get_current_site():
     return getattr(request, 'site', None)
 
 
+def get_current_site_default():
+    """
+    Return current site - falling back to the default site if not in the context of a request.
+
+    Returns:
+         (django.contrib.sites.models.Site): returns current (or default) site
+    """
+    site = get_current_site()
+    if site is None:
+        return Site.objects.get(id=settings.SITE_ID)
+    return site
```

Comment:
You don't need this function. We are currently [monkey-patching](https://github.com/edx/edx-django-sites-extensions/blob/master/django_sites_extensions/models.py#L25) the Django version of `django.contrib.sites.models.SiteManager.get_current` to provide this desired functionality. You can remove this and replace all references with `Site.objects.get_current()`.